python/port/mod/os/modos_table.c

```diff
@@ -1,16 +1,29 @@
 #include "modos.h"
 
+#include <string.h>
+#include <py/obj.h>
+#include <py/objstr.h>
+
 MP_DEFINE_CONST_FUN_OBJ_0(modos_uname_obj, modos_uname);
+MP_DEFINE_CONST_FUN_OBJ_0(modos_getlogin_obj, modos_getlogin);
 MP_DEFINE_CONST_FUN_OBJ_1(modos_remove_obj, modos_remove);
 MP_DEFINE_CONST_FUN_OBJ_2(modos_rename_obj, modos_rename);
 MP_DEFINE_CONST_FUN_OBJ_0(modos_listdir_obj, modos_listdir);
 
+#ifdef OMEGA_VERSION
+STATIC const MP_DEFINE_STR_OBJ(modos_sysname_obj, "Omega");
+#else
+STATIC const MP_DEFINE_STR_OBJ(modos_sysname_obj, "Epsilon");
+#endif
```

Comment:
This check seems useless, as this module isn't even present in Epsilon, and as we assume, since this is Omega, we won't ever not define OMEGA_VERSION.

Reply:
https://github.com/Omega-Numworks/Omega/pull/544/commits/25b74ea2f8960ecfe7f6714cc358b36f9df11c3c resolves this